File: backend/src/main/java/com/drip/admin/modules/system/vo/OnlineUserVo.java
package com.drip.admin.modules.system.vo;

public record OnlineUserVo(
    String tokenId,
    Long userId,
    String username,
    String realName,
    String deviceType,
    String ip,
    String userAgent,
    String loginAt,
    String lastActiveAt
) {
}

File: backend-go/internal/modules/system/vo/online_user_vo.go
package vo

import (
	"drip-admin/backend-go/internal/common"
)

type OnlineUserVo struct {
	TokenID      string             `json:"tokenId"`
	UserID       common.Int64String `json:"userId"`
	Username     string             `json:"username"`
	RealName     string             `json:"realName"`
	DeviceType   string             `json:"deviceType"`
	IP           string             `json:"ip"`
	UserAgent    string             `json:"userAgent"`
	LoginAt      string             `json:"loginAt"`
	LastActiveAt string             `json:"lastActiveAt"`
	ExpireAt     string             `json:"expireAt"`
	Current      bool               `json:"current"`
}
